extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -6,7 +6,7 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 	'wikia.document',
 	'wikia.log',
 	'wikia.window'
-], function (googleSlots, helper, doc, log, window) {
+], function (googleSlots, recovery, doc, log, window) {
```

Comment:
👍  for renaming `helper` to `recovery` - how about renaming the file as well? (helper.js to recovery.js)